### DISCUSSION TITLE
Reading nwb filed with 1.0.6 scheme

### DISCUSSION THREAD

**Discussion by kristinaste** [NeurodataWithoutBorders/helpdesk#31] [Q&A]
Hi, I am trying to read a .nwb file created with 1.0.6 nwb scheme. As long as I tried, matlab NWB extension only supports the files starting with 2.0.1 scheme. I made a virtual environment with Python 2.7 with hdmf v.1.3.3 and pynwb version 0.6.0 installed but I keep getting the error " builder 'root' does not have a data_type". 

What would be the workflow to read the old nwb files and possibly convert them to the 2.X format? The only questions related to the version conversion are about 2.X format. Thanks.

**Comment by bendichter** [comment]
Hi @kristinaste!

PyNWB and MatNWB only support NWB files for versions 2+. If you want to read an NWB 1.0 file, I would recommend using an HDF5 reader like h5py.
